### DISCUSSION TITLE
Error in workload_detail_volume

### DISCUSSION THREAD

**Discussion by luddite516** [NetApp/harvest#803] [Q&A] [answered]
Just began collection workload detail data with harvest.  

I am running into an error which I cannot track down.  Every 20 minutes, I get the following error message in the log for each poller I am running:

{"level":"error","Poller":"xxxxxxxx","collector":"ZapiPerf:WorkloadDetailVolume","stack":[{"func":"New","line":"35","source":"errors.go"},{"func":"(*Matrix).addMetric","line":"176","source":"matrix.go"},{"func":"(*Matrix).NewMetricFloat64","line":"142","source":"matrix.go"},{"func":"(*ZapiPerf).PollCounter","line":"980","source":"zapiperf.go"},{"func":"(*task).Run","line":"61","source":"schedule.go"},{"func":"(*AbstractCollector).Start","line":"293","source":"collector.go"},{"func":"goexit","line":"1581","source":"asm_amd64.s"}],"error":"duplicate metric key => CPU_dblade","task":"counter","caller":"goharvest2/cmd/poller/collector/collector.go:340","time":"2022-01-26T16:57:32-05:00"}

I have not modified any of the collection templates; just added workload_detail_volume to my collection profile.

Harvest seems to be collecting the workload data just fine.  I am just asking if the above error can be eliminated or if there is something that I did incorrectly.

Any pointers if this can be fixed or is it a code issue?

Thanks!

**Comment by rahulguptajss** [comment, marked as answer]
@luddite516 Thanks. This is a code issue as you have pointed, this will not impact counter collection. We'll fix this bug in coming releases.
